### PR TITLE
DT-1312 Create end point to create POM teams in all possible prisons

### PR DESCRIPTION
**Pull request by andymarke**

### CODE REVIEW

File: src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/ProbationAreaRepository.java [hunk 1]
```diff
@@ -6,11 +6,15 @@
...
+import java.util.List;
 import java.util.Optional;
 
 public interface ProbationAreaRepository extends JpaRepository<ProbationArea, Long>, JpaSpecificationExecutor<ProbationArea> {
     Optional<ProbationArea> findByCode(String code);
 
     @Query("select pa from ProbationArea pa, RInstitution institution where pa.institution = institution and institution.nomisCdeCode = :nomisCdeCode")
     Optional<ProbationArea> findByInstitutionByNomsCDECode(@Param("nomisCdeCode") String nomsPrisonInstitutionCode);
+
+    @Query("select pa from ProbationArea pa, RInstitution institution where pa.institution = institution and institution.nomisCdeCode is not null and institution.nomisCdeCode <> 'OUT'")
```
Comment (andymarke) [inline src/main/java/uk/gov/justice/digital/delius/jpa/standard/repository/ProbationAreaRepository.java:18]: get all but exclude the special "OUT" prison, that obviously isn't a real prison

File: src/main/resources/db/schema/V1_0__schema.sql [hunk 2]
```diff
@@ -114,7 +114,7 @@ create sequence SUBJECT_ACCESS_REPORT_ID_SEQ
     ;
 
 create sequence TEAM_ID_SEQ
-    minvalue 2500000000
```
Comment (andymarke) [inline src/main/resources/db/schema/V1_0__schema.sql:117]: boost test sequence to beyond existing data

File: src/test/java/uk/gov/justice/digital/delius/util/EntityHelper.java [hunk 3]
```diff
@@ -741,8 +741,8 @@ public static ProbationArea aProbationArea() {
                 .description("NPS North East")
                 .privateSector(0L)
                 .organisation(Organisation.builder().build())
-                .providerTeams(List.of())
-                .teams(List.of())
+                .providerTeams(new ArrayList<>())
+                .teams(new ArrayList<>())
```
Comment (andymarke) [inline src/test/java/uk/gov/justice/digital/delius/util/EntityHelper.java:745]: make them mutable by default to allow for new teams

**Review by chris-faulkner-mojdt — APPROVED** [review]
LGTM

**Pull request merged**